app/models/concerns/rubygem_searchable.rb

```diff
@@ -37,9 +40,13 @@ def as_indexed_json(_options = {})
              }
 
     mapping do
-      indexes :name, analyzer: 'rubygem'
-      indexes :indexed, type: 'boolean'
-      indexes :info, analyzer: 'english'
+      indexes :name, type: 'multi_field' do
+        indexes :name, analyzer: 'rubygem'
+        indexes :suggest, type: 'completion'
```

Comment:
@karmi Is this right? I tried a few different ways and this seemed to work. I couldn't find good documentation about the syntax here.